diff --git a/node_modules/react-native-paper/lib/commonjs/components/Searchbar.js b/node_modules/react-native-paper/lib/commonjs/components/Searchbar.js
index 2375c2f..3c5698a 100644
--- a/node_modules/react-native-paper/lib/commonjs/components/Searchbar.js
+++ b/node_modules/react-native-paper/lib/commonjs/components/Searchbar.js
@@ -161,7 +161,7 @@ const Searchbar = (0, _forwardRef.forwardRef)(({
       ...font,
       ..._reactNative.Platform.select({
         web: {
-          outline: 'none'
+          outlineStyle: 'none'
         }
       })
     }, isV3 && (isBarMode ? styles.barModeInput : styles.viewModeInput), inputStyle],
diff --git a/node_modules/react-native-paper/lib/commonjs/components/TextInput/TextInputFlat.js b/node_modules/react-native-paper/lib/commonjs/components/TextInput/TextInputFlat.js
index 3a11336..001f7cf 100644
--- a/node_modules/react-native-paper/lib/commonjs/components/TextInput/TextInputFlat.js
+++ b/node_modules/react-native-paper/lib/commonjs/components/TextInput/TextInputFlat.js
@@ -314,7 +314,7 @@ const TextInputFlat = ({
       textAlign: textAlign ? textAlign : _reactNative.I18nManager.getConstants().isRTL ? 'right' : 'left',
       minWidth: Math.min(parentState.labelTextLayout.width + 2 * FLAT_INPUT_OFFSET, MIN_WIDTH)
     }, _reactNative.Platform.OS === 'web' ? {
-      outline: 'none'
+      outlineStyle: 'none'
     } : undefined, adornmentStyleAdjustmentForNativeInput, contentStyle],
     testID
   })), /*#__PURE__*/React.createElement(_TextInputAdornment.default, adornmentProps));
diff --git a/node_modules/react-native-paper/lib/commonjs/components/TextInput/TextInputOutlined.js b/node_modules/react-native-paper/lib/commonjs/components/TextInput/TextInputOutlined.js
index 56eb898..e0120d6 100644
--- a/node_modules/react-native-paper/lib/commonjs/components/TextInput/TextInputOutlined.js
+++ b/node_modules/react-native-paper/lib/commonjs/components/TextInput/TextInputOutlined.js
@@ -313,7 +313,7 @@ const TextInputOutlined = ({
       paddingHorizontal: INPUT_PADDING_HORIZONTAL,
       minWidth: Math.min(parentState.labelTextLayout.width + 2 * INPUT_PADDING_HORIZONTAL, MIN_WIDTH)
     }, _reactNative.Platform.OS === 'web' ? {
-      outline: 'none'
+      outlineStyle: 'none'
     } : undefined, adornmentStyleAdjustmentForNativeInput, contentStyle],
     testID
   })), /*#__PURE__*/React.createElement(_TextInputAdornment.default, adornmentProps));
diff --git a/node_modules/react-native-paper/lib/module/components/Searchbar.js b/node_modules/react-native-paper/lib/module/components/Searchbar.js
index b6d4c30..b0109a7 100644
--- a/node_modules/react-native-paper/lib/module/components/Searchbar.js
+++ b/node_modules/react-native-paper/lib/module/components/Searchbar.js
@@ -153,7 +153,7 @@ const Searchbar = forwardRef(({
       ...font,
       ...Platform.select({
         web: {
-          outline: 'none'
+          outlineStyle: 'none'
         }
       })
     }, isV3 && (isBarMode ? styles.barModeInput : styles.viewModeInput), inputStyle],
diff --git a/node_modules/react-native-paper/lib/module/components/TextInput/TextInputFlat.js b/node_modules/react-native-paper/lib/module/components/TextInput/TextInputFlat.js
index 6af651b..5c3d12d 100644
--- a/node_modules/react-native-paper/lib/module/components/TextInput/TextInputFlat.js
+++ b/node_modules/react-native-paper/lib/module/components/TextInput/TextInputFlat.js
@@ -307,7 +307,7 @@ const TextInputFlat = ({
       textAlign: textAlign ? textAlign : I18nManager.getConstants().isRTL ? 'right' : 'left',
       minWidth: Math.min(parentState.labelTextLayout.width + 2 * FLAT_INPUT_OFFSET, MIN_WIDTH)
     }, Platform.OS === 'web' ? {
-      outline: 'none'
+      outlineStyle: 'none'
     } : undefined, adornmentStyleAdjustmentForNativeInput, contentStyle],
     testID
   })), /*#__PURE__*/React.createElement(TextInputAdornment, adornmentProps));
diff --git a/node_modules/react-native-paper/lib/module/components/TextInput/TextInputOutlined.js b/node_modules/react-native-paper/lib/module/components/TextInput/TextInputOutlined.js
index a3b15c3..685c259 100644
--- a/node_modules/react-native-paper/lib/module/components/TextInput/TextInputOutlined.js
+++ b/node_modules/react-native-paper/lib/module/components/TextInput/TextInputOutlined.js
@@ -305,7 +305,7 @@ const TextInputOutlined = ({
       paddingHorizontal: INPUT_PADDING_HORIZONTAL,
       minWidth: Math.min(parentState.labelTextLayout.width + 2 * INPUT_PADDING_HORIZONTAL, MIN_WIDTH)
     }, Platform.OS === 'web' ? {
-      outline: 'none'
+      outlineStyle: 'none'
     } : undefined, adornmentStyleAdjustmentForNativeInput, contentStyle],
     testID
   })), /*#__PURE__*/React.createElement(TextInputAdornment, adornmentProps));
